proper_parens.py

```diff
@@ -2,6 +2,22 @@
 from __future__ import unicode_literals
 
 
+<<<<<<< HEAD
```

Comment:
You've committed a conflicted merge.  This line, line 20 and line 46 are all completely illegal in Python and will not allow tests to run.  Please fix this problem